charmcraft/config.py

```diff
@@ -150,7 +171,18 @@ def format_pydantic_errors(errors):
         formatted_loc = format_pydantic_error_location(error["loc"])
         formatted_msg = format_pydantic_error_message(error["msg"])
 
-        combined.append(f"- {formatted_msg} in field {formatted_loc!r}")
+        if formatted_msg == "field required":
```

Comment:
Could this be affected by locale?

Reply:
negative, but I wonder if we can catch / determine the error class directly, that may be better in the future